File: src/net/eduard/economy/command/EconomyAddCommand.java

package net.eduard.economy.command;

import net.eduard.economy.EduEconomy;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;
import net.eduard.api.lib.modules.Extra;
import net.eduard.api.lib.player.FakePlayer;

public class EconomyAddCommand extends CommandManager {

	public EconomyAddCommand() {
		super("add", "adicionar");
		setUsage("/money add <jogador> <quantidade>");
		setDescription("Adicionar saldo para o jogador");
	}

	@Override
	public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
		if (args.length < 3) {
			sendUsage(sender);
		} else {
			FakePlayer fakeplayer = new FakePlayer(args[1]);
			double amount = Extra.fromMoneyToDouble(args[2]);
		
			sender.sendMessage(EduEconomy.getInstance().message("money-add").replace("$player", fakeplayer.getName())
					.replace("$amount", Extra.formatMoney(amount)));
			EduEconomy.getInstance().getManager().addCoins(fakeplayer, amount);
			if (fakeplayer.getPlayer() != null) {
				fakeplayer.getPlayer().sendMessage(
						EduEconomy.getInstance().message("money-changed").replace("$player", sender.getName()));
			}
		}

		return true;
	}

}

File: src/main/java/net/eduard/economy/command/EconomyAddCommand.kt
package net.eduard.economy.command

import net.eduard.api.lib.manager.CommandManager
import net.eduard.api.lib.modules.FakePlayer
import net.eduard.api.lib.modules.Extra
import net.eduard.economy.EduEconomyPlugin
import org.bukkit.command.CommandSender
import kotlin.math.abs

class EconomyAddCommand : CommandManager("add", "adicionar") {
    override fun command(sender: CommandSender, args: Array<String>) {

        if (args.size < 2) {
            sendUsage(sender)
            return
        }
        val player = FakePlayer(args[0])
        var quantidade = Extra.fromMoneyToDouble(args[1])
        quantidade = abs(quantidade)
        EduEconomyPlugin.instance.manager.addCoins(player, quantidade)
        if (args.size >= 3) {
            if (args[2] == "-msg") {
                return
            }
        }
        sender.sendMessage(
            EduEconomyPlugin.instance.message("money-add")
                .replace("%player", player.name)
                .replace("%amount", Extra.formatMoney(quantidade))
        )
        player.sendMessage(
            EduEconomyPlugin.instance.message("money-changed")
                .replace("%player", sender.name)
        )

    }


    init {
        usage = "/money add <jogador> <quantidade>"
        description = "Adicionar saldo para o jogador"
    }
}